pkg/test/framework/components/echo/config.go

```diff
@@ -53,13 +53,14 @@ type Configurable interface {
 type VMDistro = string
 
 const (
-	UbuntuXenial VMDistro = "UbuntuXenial"
-	UbuntuJammy  VMDistro = "UbuntuJammy"
-	Debian11     VMDistro = "Debian9"
-	Centos7      VMDistro = "Centos7"
-	Rockylinux8  VMDistro = "Centos8"
-
-	DefaultVMDistro = UbuntuJammy
+	UbuntuXenial  VMDistro = "UbuntuXenial"
+	UbuntuJammy   VMDistro = "UbuntuJammy"
+	Debian11      VMDistro = "Debian9"
+	Centos7       VMDistro = "Centos7"
+	Rockylinux8   VMDistro = "Centos8"
+	CentosStream8 VMDistro = "CentosStream8"
+
+	DefaultVMDistro = CentosStream8
```

Comment:
Is there any reason to have a centos8 image?

Reply:
Nope, it is not used, but we don't use other images as well. I didn't remove them to not increase the scope of this PR.